tests/test_measurements.py

```diff
@@ -117,6 +147,38 @@ def circuit():
 
         circuit()
 
+    def test_wire_order_in_tensor_prod_observables(self, tol):
+        dev = qml.device("default.qubit", wires=3)
+
+        @qml.qnode(dev)
+        def circ(obs):
+            sub_routine(label_map=range(3))
+            return qml.expval(obs)
+
+        for obs, permuted_obs in zip(obs_lst, obs_permuted_lst):
+            assert np.allclose(circ(obs), circ(permuted_obs), atol=tol, rtol=0)
+
+    @pytest.mark.parametrize("label_map", label_maps)
+    def test_wire_label_in_tensor_prod_observables(self, label_map, tol):
```

Comment:
Is the difference between this test case and the previous one that the order of observables is fixed, but their wire labels change using parametrization?